src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,33 @@
+import Markdown from 'components/core/Markdown/Markdown';
+import { VStack } from 'components/core/Spacer/Stack';
+import { BaseM, BaseXL } from 'components/core/Text/Text';
+import styled from 'styled-components';
+
+type Props = {
+  title: string;
+  description?: string;
+  children?: React.ReactNode;
+};
+
+export function EmptyState({ title, description, children }: Props) {
+  return (
+    <VStack align="center">
+      <StyledTitle>{title}</StyledTitle>
+      {description && (
+        <StyledBody>
+          <Markdown text={description} />
+        </StyledBody>
+      )}
+      {children}
```

Comment:
Q: if the EmptyState simply renders the children that it's passed in, i wonder if it's even worth having it be passed in?

it seems like these would do almost the exact same thing:
```js
// current
<EmptyState>
  <A />
</EmptyState>

vs.

// without children as prop
<>
  <EmptyState />
  <A />
</>
```

so i think we can just remove `children` as a prop, since it's not doing much!

Reply:
```js
<VStack align="center"> <-- align center the children
  <StyledTitle>{title}</StyledTitle>
  {description && (
    <StyledBody>
      <Markdown text={description} />
    </StyledBody>
  )}
  {children}
</VStack>
```

There is a little bit different. The rendered children will be center aligned. If we didn't passed it, we might need to center manually likes


```jsx
<>
  <EmptyState />
  <VStack align='center'>
   <A />
  </Vstack>
</>
```